xarray/tests/test_variable.py

```diff
@@ -1077,6 +1077,17 @@ def test_repr(self):
         ).strip()
         assert expected == repr(v)
 
+    def test_repr_html(self):
+        v = Variable(["time", "x"], [[1, 2, 3], [4, 5, 6]], {"foo": "bar"})
+        assert hasattr(v, "_repr_html_")
+        with set_options(display_style="html"):
+            html = v._repr_html_().strip()
+        # We don't do a complete string identity since
+        # html output is probably subject to change, is long and... reasons.
+        # Just test that something reasonable was produced.
+        assert html.startswith("<div") and html.endswith("</div>")
+        assert "xarray.Variable" in html
+
```

Comment:
should this be moved into `test_formatting_html.py`?

Reply:
I thought about that... but I've got the Variable class in scope here so I don't have
to mess with any imports.

Reply:
You can just use `xr.Variable` instead. I do think this test would make more test in `test_formatting_html.py` along with all the other HTML tests. Otherwise it's likely that someone reading through those tests would not  find this one.

Reply:
OK moved to `test_formatting_html.py`